pkg/trace/api/debugger_test.go

```diff
@@ -66,6 +65,50 @@ func TestDebuggerProxyHandler(t *testing.T) {
 		receiver.debuggerProxyHandler().ServeHTTP(httptest.NewRecorder(), req)
 		assert.True(t, called, "request not proxied")
 	})
+
+	t.Run("ok_truncate_ddtags", func(t *testing.T) {
+		tooLongString := strings.Repeat("x", 10000)
+		var called bool
+		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Can you add a `defer srv.Close()` for this and the other test to ensure the test server is closed?